Add three new properties (#31967)
dotnet/docs: docs/core/project-sdk/msbuild-props.md

</PropertyGroup>
```

### DisableTransitiveFrameworkReferenceDownloads

> Set the `DisableTransitiveFrameworkReferenceDownloads` property to `true` to avoid downloading extra runtime and targeting packs that your project doesn't need.

Reviewer: ```suggestion
Set the `DisableTransitiveFrameworkReferenceDownloads` property to `true` to avoid downloading extra runtime and targeting packs that are not directly referenced by your project.
```

</PropertyGroup>
```

### UseMauiEssentials

> Set the `UseMauiEssentials` property to `true` to declare an explicit reference to a project or package that depends on MAUI Essentials. This setting ensure that your project correctly pulls in the correct known framework reference for MAUI Essentials. If your project references a project that uses MAUI Essentials but you don't set this property to `true`, you might encounter build warning `NETSDK1186`.

Author: I'm not exactly sure what MAUI Essentials refers to, but it would be good to be explicit here.

> Set the `UseMauiEssentials` property to `true` to declare an explicit reference to a project or package that depends on MAUI Essentials. This setting ensure that your project correctly pulls in the correct known framework reference for MAUI Essentials. If your project references a project that uses MAUI Essentials but you don't set this property to `true`, you might encounter build warning `NETSDK1186`.

Reviewer 2: ```suggestion
Set the `UseMauiEssentials` property to `true` to declare an explicit reference to a project or package that depends on MAUI Essentials. This setting ensures that your project pulls in the correct known framework reference for MAUI Essentials. If your project references a project that uses MAUI Essentials but you don't set this property to `true`, you might encounter build warning `NETSDK1186`.
```
